core/metacling/src/TClingCallbacks.cxx

```diff
@@ -55,7 +55,6 @@ extern "C" {
    int TCling__AutoLoadCallback(const char* className);
    int TCling__AutoParseCallback(const char* className);
    const char* TCling__GetClassSharedLibs(const char* className);
-//    int TCling__IsAutoLoadNamespaceCandidate(const char* name);
```

Comment:
What about the use at `TClingCallbacks.cxx:308`:
```
    if ( !TCling__IsAutoLoadNamespaceCandidate(NSD) )
```
?

Reply:
That’s a different overload.